plugins/org.jkiss.dbeaver.model.jdbc/src/org/jkiss/dbeaver/model/impl/jdbc/JDBCDataSource.java

```diff
@@ -284,6 +287,8 @@ protected Connection openConnection(@NotNull DBRProgressMonitor monitor, @Nullab
                 authModel.endAuthentication(container, connectionInfo, connectProps);
             }
 
+            Thread.currentThread().setContextClassLoader(originalClassLoader);
```

Comment:
Just in case, it may be better to execute it in `finally`.

Reply:
Right, thank you for the feedback! Fixed in [35676a1](https://github.com/scalar-labs/dbeaver/pull/1/commits/35676a1fcd24b8f6709a278b6f8f647ff76ea97b).